plugins/woocommerce/src/Admin/API/Reports/Variations/Controller.php

```diff
@@ -39,6 +39,8 @@ class Controller extends ReportsController implements ExportableInterface {
 	 */
 	protected $param_mapping = array(
 		'variations' => 'variation_includes',
+		'products'   => 'product_includes',
+		'categories' => 'category_includes',
```

Comment:
It seems that we don't need this line (`'categories' => 'category_includes'`) in order to fix this issue. Also I think if we add this line here, we also need to define `$params['categories']` in the `get_collection_params()` method, just like you defined the `params['products']`. Otherwise [this loop for mapping registered params and request params](https://github.com/woocommerce/woocommerce/blob/0cdb36f11de1eab39ae79061d6de1d91329a710e/plugins/woocommerce/src/Admin/API/Reports/Variations/Controller.php#L66-L74) wouldn't be reached.